src/models/tools/geometry/jxg-polygon.ts

```diff
@@ -110,8 +110,22 @@ export const polygonChangeAgent: JXGChangeAgent = {
     const parents = (change.parents || [])
                       .map(id => board.objects[id as string])
                       .filter(pt => pt != null);
-    const props = assign({ id: uuid(), hasInnerPoints: true }, change.properties);
-    return parents.length ? board.create("polygon", parents, props) : undefined;
+    const props = assign({
+      id: uuid(),
+      hasInnerPoints: true,
+      clientFillColor: "#00FF00",
+      clientSelectedFillColor: "#00FF00"
```

Comment:
I presume these color changes are related to the selection model changes?

Reply:
Exactly - when you click a polygon it now gets added to the selection, so unless I give it explicit selection colors it will use the default red/gray.